FE-feat-P태그 Text 9종 생성

leejiwoo2002 opened the pull request:
### 📕 Issue Number

related issue #5 
<br/>

### 📗 작업 내역

> 구현 내용 및 작업 했던 내역

#### CenterSmallText
<img width="438" alt="스크린샷 2020-11-25 오후 4 53 59" src="https://user-images.githubusercontent.com/55074799/100198161-d2708780-2f3e-11eb-97d0-f4f426dde773.png">

#### CenterNormalText
<img width="439" alt="스크린샷 2020-11-25 오후 4 54 09" src="https://user-images.githubusercontent.com/55074799/100198172-d7353b80-2f3e-11eb-93cb-b7a1a8b2afdf.png">

#### CenterLargeText
<img width="441" alt="스크린샷 2020-11-25 오후 4 54 16" src="https://user-images.githubusercontent.com/55074799/100198180-db615900-2f3e-11eb-8478-9d2cd74969e7.png">

#### RightSmallText
<img width="436" alt="스크린샷 2020-11-25 오후 4 54 25" src="https://user-images.githubusercontent.com/55074799/100198208-e0bea380-2f3e-11eb-8b4c-2932861d7dad.png">

#### RightNormalText
<img width="446" alt="스크린샷 2020-11-25 오후 4 54 32" src="https://user-images.githubusercontent.com/55074799/100198216-e4eac100-2f3e-11eb-9900-b8940c8524af.png">

#### RightLargeText
<img width="438" alt="스크린샷 2020-11-25 오후 4 54 40" src="https://user-images.githubusercontent.com/55074799/100198234-e9af7500-2f3e-11eb-8110-2e7f63f91670.png">

#### LeftSmallText
<img width="430" alt="스크린샷 2020-11-25 오후 4 54 47" src="https://user-images.githubusercontent.com/55074799/100198244-eddb9280-2f3e-11eb-9aaf-4fade73cd228.png">

#### LeftNormalText
<img width="427" alt="스크린샷 2020-11-25 오후 4 54 55" src="https://user-images.githubusercontent.com/55074799/100198254-f2a04680-2f3e-11eb-8bdc-9d15e2877b4f.png">

#### LeftLargeText
<img width="429" alt="스크린샷 2020-11-25 오후 4 55 06" src="https://user-images.githubusercontent.com/55074799/100198275-f92ebe00-2f3e-11eb-9cf8-59a46dfd0da6.png">

<br/>

### 📘 작업 유형

- 신규 기능 추가

<br/>

### 📝 PR 특이 사항

> PR을 볼 때 주의깊게 봐야하거나 말하고 싶은 점

- 커밋 daec272 이후부터 보시면 됩니다 클릭 -> [File Changed](https://github.com/boostcamp-2020/Project16-E-Account-Book/pull/26/files/0623cb2e0a9cad1ec2064736f843dc85e0a633ff..966c1a6bd3938e3c95e8610fcf10839beab29844)

<br/><br/>

Commit:
feat : CenterSmallText 생성

- tsx 요소 생성
- props (default)
  - fontSize (12px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (center)

Commit:
feat : CenterNormalText 생성

- tsx 요소 생성
- props (default)
  - fontSize (16px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (center)

Commit:
feat : CenterLargeText 생성

- tsx 요소 생성
- props (default)
  - fontSize (24px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (center)

Commit:
feat : RightSmallText 생성

- tsx 요소 생성
- props (default)
  - fontSize (12px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (right)

Commit:
feat : RightNormalText 생성

- tsx 요소 생성
- props (default)
  - fontSize (16px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (right)

Commit:
feat : RightLargeText 생성

- tsx 요소 생성
- props (default)
  - fontSize (24px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (right)

Commit:
feat : LeftSmallText 생성

- tsx 요소 생성
- props (default)
  - fontSize (12px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (left)

Commit:
feat : LeftNormalText 생성

- tsx 요소 생성
- props (default)
  - fontSize (16px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (left)

Commit:
feat : LeftLargeText 생성

- tsx 요소 생성
- props (default)
  - fontSize (24px)
  - fontWeight (normal)
  - color (color.primary.kakaoBlack)
  - text-align (left)

do02reen24 commented on client/src/components/atoms/p/CenterLargeText/CenterLargeText.tsx:
```diff
@@ -0,0 +1,39 @@
+}
+
+const defaultProps = {
+  color: myColor.primary.kakaoBlack,
```
색상 명칭을 kakaoBlack로 네이밍한 이유가 있나요?

leejiwoo2002 replied:
카카오 공식홈페이지에서 찾았었는데 지금 링크를 못찾겠네요 ㅠㅠ
https://www.kakaocorp.com/kakao/introduce/ci

이런식으로 CI 소개해주는 페이지였는데 대표 색으로 노란색계열, 검은색계열, 흰색, 빨간색 이렇게 4종류 있던것같습니다
그래서 저렇게 이름을 붙였어요!

do02reen24 replied:
저희 프로젝트에서 사용할 때 덜 와닿는 것 같아서 네이밍을 바꾸면 어떨까요?

leejiwoo2002 replied:
제시 해주시면 따라가겠습니다!

do02reen24 replied:
`black`으로 그냥 가면 어떨까요? `myColor.primary`를 앞에 붙이기 때문에 `black`으로 진행하셔도 괜찮을 것 같아요!

do02reen24 reviewed (commented):
전체적으로 통일성 있게 잘 짜여진 것 같습니다😀👍

leejiwoo2002 commented:
> 전체적으로 통일성 있게 잘 짜여진 것 같습니다😀👍

컨C컨V의 힘

JunYoung7 commented on client/src/components/atoms/p/RightNormalText/RightNormalText.tsx:
```diff
@@ -0,0 +1,39 @@
+
+const defaultProps = {
+  color: myColor.primary.kakaoBlack,
+  fontSize: '16px',
```
fontSize 정해두신 기준이나 규격이 있나요??🙋‍♂️

leejiwoo2002 replied:
저거 컴포넌트 엑셀에 정해뒀던 값으로 일단 채워뒀습니다!

JunYoung7 reviewed (approved):
고생하셨습니다 지우님!

Commit:
fix : color.kakaoBlack 옵션 수정

- primary.kakaoBlack -> primary.black으로 이름 변경
- 이름 변경에 따라, 사용중이던 요소들에서 호출 이름 변경

Zigje9 reviewed (commented):
P태그를 아톰으로 분리했을때, 정해놓은 weight, size, color 등을 사용하려고 만드는 것인데 default가 있긴 하지만 각 컴포넌트 마다 prop를 넘겨주는 것이 필요할까요? 모두 분리한 의미가 조금 퇴색 되지 않는 걸까요?

Commit:
fix : storybook 경로 대소문자 수정

leejiwoo2002 committed:
Merge branch 'develop' into FE-feat-PText
